src/json/members/yuclin.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Yu-Hsin Lin",
+  "pennkey": "yuclin",
+  "major": "Integrated Product Design",
+  "school": "Penn Engineering",
+  "bio": "Exploring Philly and bubble teas!",
+  "hometown": "Taipei, Taiwan",
+  "team": "Design",
+  "roles": [Product Designer],
+  "photo": "https://imgur.com/a/ZGPYijg",
```

```suggestion
  "photo": "https://i.imgur.com/UFO71vO.png",
```

Links to imgur.com directly don't play well with our website unfortunately - I've slotted in the link to the actual image file instead.